src/QMCWaveFunctions/BsplineFactory/SplineC2CAdoptor.h

```diff
@@ -515,9 +516,18 @@ struct SplineC2CSoA: public SplineAdoptorBase<ST,3>
   {
     const PointType& r=P.activeR(iat);
     PointType ru(PrimLattice.toUnit_floor(r));
-    spline2::evaluate3d_vgh(SplineInst->spline_m,ru,myV,myG,myH);
-    assign_vgh(r,psi,dpsi,grad_grad_psi);
-    //missing
+
+    #pragma omp parallel
+    {
+      int first, last;
+      FairDivideAligned(myV.size(), getAlignment<ST>(),
+                        omp_get_num_threads(),
+                        omp_get_thread_num(),
+                        first, last);
+
+      spline2::evaluate3d_vgh(SplineInst->spline_m,ru,myV,myG,myH,first,last);
+      assign_vgh(r,psi,dpsi,grad_grad_psi,first/2,last/2);
```

Comment:
C2R and C2C have the complex splines in double sized real arrays. When the values are put into the final destination. assign_XXX put translation symmetry and works on the actual splines instead of unpacked arrays. When do you think will be a good place to add this info? In the beginning of file?

Reply:
Ok that makes sense, and I should have guessed. But it's worth documenting. Since it effects all the evaluates I think top of file in the long docstring for the class.